docs/Velox.md

```diff
@@ -76,6 +76,20 @@ Hadoop Yarn mode is supported. Note libhdfs3 is used to read from HDFS, all its
 sudo apt install -y libiberty-dev libxml2-dev libkrb5-dev libgsasl7-dev libuuid1 uuid-dev
 ```
 
+## High-Bandwidth Memory (HBM) support
+
+Gluten supports allocating memory on HBM. This feature is optional and is disabled by default. It requires [Memkind library](http://memkind.github.io/memkind/) to be installed. Please follow memkind's [readme](https://github.com/memkind/memkind#memkind) to build and install all the dependencies and the library. 
+
+To enable this feature in Gluten, users only need to add `-Denable_hbm=ON` to the maven build command. Here's an example:
```

Comment:
Please add this configuration to https://github.com/oap-project/gluten/blob/main/docs/GlutenUsage.md